opennms-services/src/test/java/org/opennms/netmgt/collectd/SnmpCollectorWithPointerLikeMibProperties.java

```diff
@@ -85,9 +87,8 @@
 })
 @JUnitConfigurationEnvironment(systemProperties = {
         "org.opennms.rrd.storeByGroup=false",
-        "db.adminPassword=postgres"
 })
-@JUnitTemporaryDatabase(reuseDatabase = false, useExistingDatabase = "testjfcopennms")
+@JUnitTemporaryDatabase(reuseDatabase = false)
 // Relies on records created in @Before so we need a fresh database for each test
 public class SnmpCollectorWithPointerLikeMibProperties implements InitializingBean, TestContextAware {
```

Comment:
The name of a `JUnit` test should end on "IT" in order to be executed by our automation tool, otherwise the test class will be ignored. Please rename the class to be `SnmpCollectorWithPointerLikeMibPropertiesIT`